flink-shaded-hadoop-2/pom.xml

```diff
@@ -0,0 +1,977 @@
+							<groupId>tomcat</groupId>
+							<artifactId>jasper-runtime</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>javax.servlet.jsp</groupId>
+							<artifactId>jsp-api</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>com.sun.jersey.jersey-test-framework</groupId>
+							<artifactId>jersey-test-framework-grizzly2</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>com.sun.jersey.jersey-test-framework</groupId>
+							<artifactId>jersey-test-framework-core</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>com.sun.jersey</groupId>
+							<artifactId>jersey-grizzly2</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish.grizzly</groupId>
+							<artifactId>grizzly-http</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish.grizzly</groupId>
+							<artifactId>grizzly-framework</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish.grizzly</groupId>
+							<artifactId>grizzly-http-server</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish.grizzly</groupId>
+							<artifactId>grizzly-rcm</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish.grizzly</groupId>
+							<artifactId>grizzly-http-servlet</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>org.glassfish</groupId>
+							<artifactId>javax.servlet</artifactId>
+						</exclusion>
+						<exclusion>
+							<groupId>com.sun.jersey.contribs</groupId>
+							<artifactId>jersey-guice</artifactId>
+						</exclusion>
+						<!--We have to exclude beanutils because it is overlapping with commons-collections-->
+						<exclusion>
+							<groupId>commons-beanutils</groupId>
+							<artifactId>commons-beanutils</artifactId>
+						</exclusion>
+					</exclusions>
+				</dependency>
+			</dependencies>
+		</profile>
+	</profiles>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>dependency-convergence</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<skip>true</skip>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<!-- Relocate the Hadoop's Guava dependency into a different namespace and put Hadoo into a fat-jar.-->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<executions>
+					<execution>
+						<!--Disable shade-flink execution to avoid duplicate relocation of dependencies-->
+						<id>shade-flink</id>
+						<phase>none</phase>
+					</execution>
+					<execution>
+						<id>shade-hadoop</id>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+						<configuration>
+							<shadedArtifactAttached>false</shadedArtifactAttached>
```

Comment:
This is the default, so we can omit it like in other modules.

Reply:
Yes, that's true.